replicatedhq/ship
# stub out --raw command some more

What I Did
------------

stub out --raw command some more. Still panics because empty implementation of the lifecycle step.

How I Did it
------------

Added Kustomize fields to `ship.Ship` to know which mode we're operating in -- it might make sense to make a separate object instead of all these if branches.

Added a branch to check `IsKustomize` and `KustomizeRaw` on `ship.Ship`, and if set, escape hatch and hard code a release lifecycle.

I put this in ship/ship.go for now, but I really thing all this stuff should be
pushed down into specs and specs.Resolver logic, just didn't want to step on any toes.

This needs a unittest, but will hold off until this gets moved in there.



How to verify it
------------

Run with --raw, observe sweet panic

Description for the Changelog
------------

This is still half built, so no changelog yet





:ship:
